tests/marqo_test.py

```diff
@@ -262,6 +269,43 @@ def setUpClass(cls) -> None:
                             "type": "no_model",
                             "dimensions": 512
                         }
+                    },
+                    {
```

Comment:
Unstructured is fine here for open source tests, we just need to adjust the logic for cloud since it only has structured.

Reply:
I'll remove it for now add add again in the future!